src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less

```diff
@@ -0,0 +1,203 @@
+                            color: @color-os-white;
+                        }
+                    }
+                }
+            }
+
+            .area-first {
+                p {
+                    font-family: @font-family__serif;
+                    font-style: @font-style__emphasis;
+                    font-weight: @font-weight__heavy;
+                }
+            }
+
+            .area-third {
+                ul li a {
+                    color: @color-os-lime;
+                    font-family: @font-family__serif;
+                    text-decoration: none;
+
+                    &:hover {
+                        color: @color-os-olive;
+                    }
+                }
+
+                .play {
+                    .lib-vendor-prefix-display();
+
+                    .app-store,
+                    .play-store {
+                        img {
+                            width: 100%
+                        }
+                    }
+
+                    .app-store {
+                        margin-left: 5px
+                    }
+
+                    .play-store {
+                        margin-right: 5px;
+                    }
+                }
+            }
+        }
+    }
+}
+
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__s) {
+    .page-footer {
+        .pagebuilder-column-group.footer-wrapper {
+            padding: 65px 20px;
+
+            .area-first {
+                margin-bottom: 40px;
+
+                .logo-footer {
+                    margin: 0 auto 40px;
+                }
+
+                p {
+                    text-align: center;
+                    margin-bottom: 0;
+                }
+            }
+
+            .area-second,
+            .area-third {
+                text-align: center;
+
+                h4 {
+                    .lib-font-size(20);
+                    margin: 40px auto 15px;
+                }
+
+                p {
+                    .lib-font-size(14);
+                }
+
+                ul {
+                    li a {
+                        .lib-font-size(14);
+                    }
+                }
+            }
+
+            .play {
+                justify-content: space-around;
+            }
+        }
+    }
+}
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__s) {
+    .page-footer {
+        .footer.content {
+            display: none;
+        }
+
+        .pagebuilder-column-group.footer-wrapper {
+            .lib-vendor-prefix-display();
+            padding: 45px 20px;
+
+            .pagebuilder-column-line {
+                &:first-child {
+                    width: 50% !important;
+                }
+
+                &:nth-child(2),
+                &:last-child {
+                    width: 25% !important;
+                }
+            }
+
+            .area-first,
+            .area-second,
+            .area-third {
+                margin: 0 10px;
```

Comment:
Same thing when you see that this construction is taking place, think about adding one common class

Reply:
Changed!